Minimatch.Pkg/Minimatch.Pkg.nuproj

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|AnyCPU">
+      <Configuration>Debug</Configuration>
+      <Platform>AnyCPU</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|AnyCPU">
+      <Configuration>Release</Configuration>
+      <Platform>AnyCPU</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\LICENSE.txt">
+      <Link>LICENSE.txt</Link>
+    </Content>
+    <Content Include="..\README.md">
+      <Link>README.md</Link>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Minimatch.Portable.Net40\Minimatch.Portable.Net40.csproj" />
+    <ProjectReference Include="..\Minimatch.Portable.Net45\Minimatch.Portable.Net45.csproj" />
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>d99cdc4a-f8de-451b-8ce7-1a3b2bd25045</ProjectGuid>
+  </PropertyGroup>
+  <PropertyGroup>
+    <NuProjPath Condition=" '$(NuProjPath)' == '' ">$(MSBuildExtensionsPath)\NuProj\</NuProjPath>
+  </PropertyGroup>
+  <Import Project="$(NuProjPath)\NuProj.props" Condition="Exists('$(NuProjPath)\NuProj.props')" />
+  <PropertyGroup Label="Configuration">
+    <Id>Minimatch</Id>
+    <Version>1.2.0</Version>
```

Comment:
It's a limitation of NuProj. However, to make it usable from a build server, I can add the following lines to the project file:

```
  <PropertyGroup>
    <VersionDependsOn>$(VersionDependsOn);GetNuPkgVersion</VersionDependsOn>
  </PropertyGroup>
  <Target Name="GetNuPkgVersion" Condition="'$(NuGetPackageVersion)' != ''">
    <PropertyGroup>
      <Version>$(NuGetPackageVersion)</Version>
    </PropertyGroup>
  </Target>
```

This would allow one to set ```NuGetPackageVersion``` which would override the version in the NuGet project.

However, I found that the version detection of NuGet sometimes doesn't work and prefer setting the version explicitly.